wsdl/ver10/search.wsdl

```diff
@@ -444,7 +446,143 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
+						</xs:element>
+						<xs:element name="KeepAliveTime" type="xs:duration">
+							<xs:annotation><xs:documentation>The time the search session will be kept alive after responding to this and subsequent requests.</xs:documentation></xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="FindImagebyNLResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="SearchToken" type="tt:JobToken">
+							<xs:annotation><xs:documentation>A unique reference to the search session created by this request.</xs:documentation></xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>	
+
+			<!-- Define GetNLSearchResults -->
+			<xs:element name="GetNLSearchResultsRequest">
+				<xs:annotation>
+					<xs:documentation>Gets results from a particular search session.</xs:documentation>
+				</xs:annotation>			
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="SearchToken" type="tt:JobToken">
+							<xs:annotation><xs:documentation>The search session to get results from.</xs:documentation></xs:annotation>
+						</xs:element>
+						<xs:element name="MinResults" type="xs:int" minOccurs="0">
+							<xs:annotation><xs:documentation>The minimum number of results to return in one response.</xs:documentation></xs:annotation>
+						</xs:element>
+						<xs:element name="MaxResults" type="xs:int" minOccurs="0">
+							<xs:annotation><xs:documentation>The maximum number of results to return in one response.</xs:documentation></xs:annotation>
+						</xs:element>
+						<xs:element name="WaitTime" type="xs:duration" minOccurs="0">
+							<xs:annotation><xs:documentation>The maximum time before responding to the request, even if the MinResults parameter is not fulfilled.</xs:documentation></xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="GetNLSearchResultsResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="ResultList" type="tt:FindImageResultList" />
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+
+			<!-- Define FindImagebyImage -->
+			<xs:element name="FindImagebyImageRequest">
+				<xs:annotation>
+					<xs:documentation>Starts a search session and specifies the search parameters.</xs:documentation>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="StartPoint" type="xs:dateTime">
+							<xs:annotation><xs:documentation>Start time for the search.</xs:documentation></xs:annotation>
+						</xs:element>
+						<xs:element name="EndPoint" type="xs:dateTime">
+							<xs:annotation><xs:documentation>End time for the search.</xs:documentation></xs:annotation>
+						</xs:element>
+						<xs:element name="RecordingToken" type="tt:RecordingReference" minOccurs="0" maxOccurs="unbounded">
```

Comment:
add maxOccurs="unbounded" which will allow more than one recording container to search.